docs/variants/protectli_vp2420/releases.md

```diff
@@ -0,0 +1,52 @@
+# Release Notes
+
+Following Release Notes describe status of open-source firmware development for
+Protectli VP2420 family.
+
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+<center>
+[Subscribe to Protectli VP2420 Dasharo Release Newsletter]
+[newsletter]{.md-button .md-button--primary .center}
+</center>
+
+Test results for this platform can be found
+[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=975611333).
+
+## v1.0.0 - 2022-TBD
```

Comment:
Before merging we should add the proper date in this place

Reply:
Added today `2022-12-22` as the date.